i18n/da.json

```diff
@@ -0,0 +1,580 @@
+    "campaigns.dateAndTime": "Dato og tid",
+    "campaigns.ended": "Afslutet",
+    "campaigns.errorSendTest": "Fejl under udsendelse af test: {error}",
+    "campaigns.fieldInvalidBody": "Fejl under kompilering af kampagne-hoveddel: {error}",
+    "campaigns.fieldInvalidFromEmail": "Ugyldig `fra_email`.",
+    "campaigns.fieldInvalidListIDs": "Ugyldig liste ID'er.",
+    "campaigns.fieldInvalidMessenger": "Ukendt besked {name}.",
+    "campaigns.fieldInvalidName": "Ugyldig længde for navn.",
+    "campaigns.fieldInvalidSendAt": "Planlagt dato bør være i fremtiden.",
+    "campaigns.fieldInvalidSubject": "Ugyldig længde på emne.",
+    "campaigns.formatHTML": "Format HTML",
+    "campaigns.fromAddress": "Fra adresse",
+    "campaigns.fromAddressPlaceholder": "Dit navn <noreply@yoursite.com>",
+    "campaigns.invalid": "Ugyldig kampagne",
+    "campaigns.invalidCustomHeaders": "Ugyldig tilpassede headere: {error}",
+    "campaigns.markdown": "Markdown",
+    "campaigns.needsSendAt": "Kampagnen behøver en dato for at kunne planlægges.",
+    "campaigns.newCampaign": "Ny kampagne",
+    "campaigns.noKnownSubsToTest": "Ingen kendt abonnent til test.",
+    "campaigns.noOptinLists": "Ingen tilmeldt-lister fundet til at oprette kampagne.",
+    "campaigns.noSubs": "Der er ingen abonnenter i den valgte liste til at oprette kampagnen.",
+    "campaigns.noSubsToTest": "Der er ingen abonnenter at sende til",
+    "campaigns.notFound": "Kampagne ikke fundet",
+    "campaigns.onlyActiveCancel": "Kun aktive kampagner kan annulleres.",
+    "campaigns.onlyActivePause": "Kun aktive kampagner kan sættes på pause.",
+    "campaigns.onlyDraftAsScheduled": "Kun udkast til kampagner kan planlægges.",
+    "campaigns.onlyPausedDraft": "Kun kampagner og kladder, der er sat på pause, kan startes.",
+    "campaigns.onlyScheduledAsDraft": "Kun planlagte kampagner kan gemmes som kladder.",
+    "campaigns.pause": "Pause",
+    "campaigns.plainText": "Almindelig tekst",
+    "campaigns.preview": "Forhåndsvisning",
+    "campaigns.progress": "Fremskridt",
+    "campaigns.queryPlaceholder": "Navn eller emne",
+    "campaigns.rateMinuteShort": "min",
+    "campaigns.rawHTML": "Rå HTML",
+    "campaigns.removeAltText": "Fjern alternativ almindelig tekstbesked",
+    "campaigns.richText": "RTf",
+    "campaigns.schedule": "Planlæg kampagne",
+    "campaigns.scheduled": "Planlagt",
+    "campaigns.send": "Sende",
+    "campaigns.sendLater": "Send senere",
+    "campaigns.sendTest": "Send testmeddelelse",
+    "campaigns.sendTestHelp": "Tryk på Enter efter at have indtastet en adresse for at tilføje flere modtagere. Adresserne skal tilhøre eksisterende abonnenter.",
+    "campaigns.sendToLists": "Lister, der skal sendes til",
+    "campaigns.sent": "Sendt",
+    "campaigns.start": "Start kampagne",
+    "campaigns.started": "\{name}\" startet",
```

Comment:
This, and a few instances below, have broken quotes. Should be `"\"{name}\" startet"`

Reply:
Let me check